### PR TITLE
fix: add back user interfacing imports in init.py

### PR DESCRIPTION
**Pull request by Tengxu-Sun**

### CODE REVIEW

**Comment by NOBLES5E** [comment]
@Tengxu-Sun see https://flake8.pycqa.org/en/latest/user/violations.html for how to ignore flake8 violations intentionally

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

File: bagua/torch_api/__init__.py [hunk 1]
```diff
@@ -2,14 +2,24 @@
...
-from .communication import init_process_group
 from .distributed import bagua_init
 from .env import (
     get_rank,
     get_world_size,
     get_local_rank,
     get_local_size,
 )
+from .communication import (  # noqa: F401
+    is_initialized,
```
Comment (NOBLES5E) [inline bagua/torch_api/__init__.py:13]: is this needed by user?

File: bagua/torch_api/__init__.py [hunk 2]
```diff
@@ -2,14 +2,24 @@
...
+    init_process_group,
+    broadcast_coalesced,
+    broadcast,
+    allreduce_coalesced,
+    allreduce,
+    get_bagua_hyperparameters,
+    get_hyperparameters_service_client,
```
Comment (NOBLES5E) [inline bagua/torch_api/__init__.py:20]: are these used by user?

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

File: bagua/torch_api/__init__.py (hunk 2 above)
Comment (Tengxu-Sun) [reply bagua/torch_api/__init__.py:20]: not totally, ok, i will check it

Comment (Tengxu-Sun) [reply bagua/torch_api/__init__.py:20]: not totally, ok, i will check it

**Pull request merged**